ppr-ui/src/components/mhrRegistration/YourHome/ManufacturerMakeModel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
```

Comment:
Are all these helpers required?

Reply:
Can be shortened to `py` here as well

Reply:
I copied them over from another card. Shortened.